app/gulpfile.js

```diff
@@ -67,11 +78,17 @@ function bundleShare(b, name) {
     });
 }
 
-gulp.task('dev', ['serve', 'buildAdmin', 'buildMap', 'watch'], function(){
-    console.log('Starting dev environnement');
+gulp.task('admin-dev', ['serve-admin', 'buildAdmin', 'watch-admin'], function(){
+    console.log('Starting admin in dev');
+});
+
+gulp.task('admin-prod', ['serve-admin', 'buildAdmin']);
+
+gulp.task('app-dev', ['serve-app', 'buildMap', 'watch-app'], function(){
+    console.log('Starting app in dev');
 });
 
-gulp.task('prod', ['serve', 'buildAdmin', 'buildMap']);
+gulp.task('app-prod', ['serve-app', 'buildMap']);
 
-gulp.task('default', ['serve', 'buildAdmin', 'buildMap', 'watch']);
+gulp.task('default', ['serve-app', 'serve-prod', 'buildAdmin', 'buildMap', 'watch']);
```

Comment:
Per discussion, we chose App (I'll change it in a commit to this PR).

`serve-prod` is a typo (should be `serve-admin`). Same for `watch`.